File: bootstrap/src/main/java/net/dontdrinkandroot/wicket/bootstrap/headeritem/BootstrapCssHeaderItem.java
package net.dontdrinkandroot.wicket.bootstrap.headeritem;

import net.dontdrinkandroot.wicket.bootstrap.css.BootstrapCssResourceReference;

import org.apache.wicket.markup.head.CssReferenceHeaderItem;


public class BootstrapCssHeaderItem extends CssReferenceHeaderItem {

	public BootstrapCssHeaderItem() {

		super(new BootstrapCssResourceReference(), null, "screen", null);
	}

}

File: bootstrap/src/main/kotlin/net/dontdrinkandroot/wicket/bootstrap/headeritem/BootstrapCssHeaderItem.kt
package net.dontdrinkandroot.wicket.bootstrap.headeritem

import org.apache.wicket.markup.head.CssUrlReferenceHeaderItem
import org.apache.wicket.markup.html.CrossOrigin

class BootstrapCssHeaderItem : CssUrlReferenceHeaderItem(
    "https://cdn.jsdelivr.net/npm/bootstrap@5$BOOTSTRAP_VERSION/dist/css/bootstrap.min.css",
    null,
    "stylesheet"
) {

    init {
        integrity = "sha384-giJF6kkoqNQ00vy+HMDP7azOuL0xtbfIcaT9wjKHr8RbDVddVHyTfAAsrekwKmP1"
        crossOrigin = CrossOrigin.ANONYMOUS
    }

    override fun getDependencies() = mutableListOf(BootstrapJsHeaderItem())
}